packages/react-vega/package.json

```diff
@@ -18,14 +18,18 @@
     "types"
   ],
   "dependencies": {
-    "prop-types": "^15.7.2"
+    "prop-types": "^15.7.2",
+    "vega-embed": "^4.0.0"
   },
   "peerDependencies": {
     "react": "^16.0.0",
     "vega": "^5.3.4"
   },
+  "devDependencies": {
+    "vega": "^5.3.4"
```

Comment:
not necessary because this is already in the root `package.json` so when you `yarn install` this monorepo will have `vega` for development